### PR TITLE
Merging note-it app code from main to production

### PR DESCRIPTION
**Pull request by karleene**
- added my firebase, header, footer, quotes, form, and notes components
- added my functions (handleSubmit, handleDelete, handleSave), useEffect, useState, axios call, and date variable in app.js

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,156 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="app">
+      <Header />
```
Comment (joeycsea) [inline src/App.js:134]: Maybe add the < wrapper > div here so you don't have to repeat it in all the other component's return

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,156 @@
...
+import axios from 'axios';
+import Header from './Header';
+import Quote from './Quote';
+import Form from './Form';
+import Notes from './Notes';
+import Footer from './Footer';
+import { useState, useEffect } from 'react';
 
 function App() {
+
```
Comment (joeycsea) [inline src/App.js:13]: I like that you have all your states and functions in App.js. Very well organized and easy to follow

**Review by joeycsea — COMMENTED** [review]
Good job Karleen!!

File: src/App.js (hunk 2 above)
Comment (PauCodes) [inline src/App.js:13]: Same as Joey, very well organized :) 
Remember to delete all your comments before deploying from production.

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,156 @@
...
+  const [quoteOfTheDay, setQuoteOfTheDay] = useState({});
+
+  // useState to keep track of the notes added / deleted
+  const [notes, setNotes] = useState([]);
+
+  // useState to keep track of what the user inputs
+  const [userInput, setUserInput] = useState('');
+
+  // on initial render/ component mount, useEffect to fetch the api for a random quote of the day to show on the page
+  useEffect(() => {
```
Comment (PauCodes) [inline src/App.js:28]: Nice job using both, firebase and api!

File: src/Header.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+// Return an H1 to show on the page
+
```
Comment (PauCodes) [inline src/Header.js:2]: I like how you used components for  each functionality. Very well organized!!

**Review by PauCodes — COMMENTED** [review]
Great job Karleen!! Great idea

File: src/App.js [hunk 5]
```diff
@@ -1,25 +1,156 @@
...
+      const note = { 
+        'message': userInput, 
+        'date': date,
+      }
+      push(dbRef, note);
+    } else {
+      alert('Please enter a note')
+    }
+
+    window.scrollBy({ top: 200, behavior: "smooth" })
```
Comment (natalielaing) [inline src/App.js:100]: Love this! I'm going to add something like this to my own code!

**Pull request merged**